### DISCUSSION TITLE
About Mountpoint

### DISCUSSION THREAD

**Discussion by HamzaCetin0** [semuconsulting/pygnssutils#72] [General]
I have a question about the mountpoint. When I changed in the mountpoint from "pygnssutils" to any other mountpoint I could not connect to the server and cant get any RTCM data. Is there a way to set a custom mountpoint to stream the RTCM data?

**Comment by semuadmin** [comment]
Assuming you're referring to the `gnssserver` CLI utility in NTRIP mode, this only supports a single mountpoint named 'pygnssutils'. It is not possible to rename this.

Why would you want to set a custom mountpoint name?

**Reply by HamzaCetin0** [reply]
Sorry, no reason. I think ı just misunderstood. Thank you for that. Ihave one last question. I am using a raspberry pi to act as NTRIP caster. This raspberry pi will be a base station and will work constantly. So,  in case of a shutdown of the raspberry pi, I want to run "gnssserver --inport .." command at every reboot of the raspberry pi. I use crontab to do that but it didnt work. So do you know how to do that?

**Reply by noob3-3** [reply]
I can write a python script, start the NTRIP broadcaster, use systemctl to do self-starting, if I need to give my specific implementation
